app/controllers/application_controller.rb

```diff
@@ -1,7 +1,10 @@
 class ApplicationController < ActionController::Base
-  helper_method :current_user
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
- [ ] @Team Great work setting up devise for your app, however you forgot to ensure users login in to the app are authenticated, currently I don't need to login to access any page, kindly consider adding user authentication in your app
```suggestion
  before_action :configure_permitted_parameters, if: :devise_controller?
  before_action :authenticate_user!
```